src/components/LoginRegister/index.js

```diff
@@ -0,0 +1,185 @@
+            shrink: true,
+          }}
+          placeholder={formatMessage({ id: 'webHome.emailAddress' })}
+          variant="outlined"
+        />
+        <InputLabel>{formatMessage({ id: 'webHome.password' })}</InputLabel>
+        <TextField
+          id="passwordRegister"
+          type="password"
+          fullWidth
+          margin="normal"
+          InputLabelProps={{
+            shrink: true,
+          }}
+          placeholder={formatMessage({ id: 'webHome.myPassword' })}
+          variant="outlined"
+        />
+        <InputLabel>{formatMessage({ id: 'webHome.confirmPassword' })}</InputLabel>
+        <TextField
+          id="confirmPasswordRegister"
+          type="password"
+          fullWidth
+          margin="normal"
+          InputLabelProps={{
+            shrink: true,
+          }}
+          placeholder={formatMessage({ id: 'webHome.repeatPassword' })}
+          variant="outlined"
+        />
+      </div>
+    );
+  }
+
+  /**
+  * View fields login
+  */
+  viewFieldsLogin() {
+    const { formatMessage } = this.props.intl;
+    return (
+      <div className="webapp-login">
+        <Typography variant="h5">
+          <FormattedMessage id="webHome.login"></FormattedMessage>
+        </Typography>
+        <br />
+        <InputLabel>{formatMessage({ id: 'webHome.email' })}</InputLabel>
+        <TextField
+          id="email"
+          fullWidth
+          type="email"
+          margin="normal"
+          InputLabelProps={{
+            shrink: true,
+          }}
+          placeholder={formatMessage({ id: 'webHome.emailAddress' })}
+          variant="outlined"
+
+        />
+        <InputLabel>{formatMessage({ id: 'webHome.password' })}</InputLabel>
+        <TextField
+          id="password"
+          type="password"
+          fullWidth
+          margin="normal"
+          InputLabelProps={{
+            shrink: true,
+          }}
+          placeholder={formatMessage({ id: 'webHome.myPassword' })}
+          variant="outlined"
+        />
+        <div className="forgot-password">
+          <Button onClick={this.onPressForgotPassword}>
+            <FormattedMessage id="webHome.forgotPassword"></FormattedMessage>
+          </Button>
+        </div>
+      </div>
+    );
+  }
+
+  /**
+  *  On press login
+  * This function is called by press the button login
+  */
+  onPressLogin() {
+  }
+
+  /**
+  * On press create account
+  * This function is called by press the button create account
+  */
+  onPressCreateAccount() {
+  }
+
+  /**
+  * On press forgot password
+  * This function is called by press the link forgot password
+  */
+  onPressForgotPassword() {
+  }
+
+  render() {
+    return (
+      <div className="webapp-login">
+        <form noValidate autoComplete="off">
+          <Grid container spacing={6}>
+            <Grid item xs={6}>
+              {this.viewFieldsLogin()}
+            </Grid>
+            <Grid item xs={6}>
+              {this.viewFieldsRegister()}
```

Comment:
Please use jsx syntax, to do that you need change name function o PascalCase and using like
`<this.ViewFieldsRegister /> `

Reply:
done! I had used the PascalCase notation for the View functions, please let me know if you want me to change also the functions: onPressFacebook, onPressGoogle, onPressApple, onPressLogin, onPressRegister